legal-api/tests/unit/services/filings/validations/test_restoration.py

```diff
@@ -197,16 +197,16 @@ def test_validate_relationship(session, test_status, restoration_type, expected_
         ('invalid_lesser', 'limitedRestoration', relativedelta(days=25), False),
         ('missing', 'limitedRestoration', None, False),
 
-        ('greater', 'limitedRestorationExtension', relativedelta(years=3), True),
-        ('invalid_greater', 'limitedRestorationExtension', relativedelta(years=3, days=1), False),
+        ('greater', 'limitedRestorationExtension', relativedelta(years=2), True),
+        ('invalid_greater', 'limitedRestorationExtension', relativedelta(years=2, days=1), False),
         ('lesser', 'limitedRestorationExtension', relativedelta(months=1), True),
         ('invalid_lesser', 'limitedRestorationExtension', relativedelta(days=25), False),
         ('missing', 'limitedRestorationExtension', None, False),
     ]
 )
 def test_validate_expiry_date(session, test_name, restoration_type, delta_date, is_valid):
     """Assert that expiry date is validated."""
-    business = Business(identifier='BC1234567', legal_type='BC')
+    business = Business(identifier='BC1234567', legal_type='BC', restoration_expiry_date=LegislationDatetime.now())
```

Comment:
Add test to consider above comment (by setting `restoration_expiry_date` to None when `restoration_type` is `limitedRestoration`)